barretenberg/cpp/src/barretenberg/vm/generated/avm_circuit_builder.cpp

```diff
@@ -318,6 +324,10 @@ template <typename FF> std::vector<std::string> AvmFullRow<FF>::names()
              "lookup_byte_lengths",
              "lookup_byte_operations",
              "lookup_opcode_gas",
+             "range_check_l2_gas_hi",
+             "range_check_l2_gas_lo",
+             "range_check_da_gas_hi",
+             "range_check_da_gas_lo",
              "kernel_output_lookup",
```

Comment:
Why are there new entries in this vector for gas?

Reply:
No idea! someone forgot to regenerate after modifying the PIL files? It shouldn't be related to my changes.

Reply:
This is related to my latest PR on range checking gas. Probably somebody/me forgot to regenerate code or overwrote my changes.